tests/lightning_base/test_measurements_class.py

```diff
@@ -136,8 +135,10 @@ def test_state_diagonalizing_gates_measurements(self, lightning_sv, mp):
             qml.prod(qml.PauliX(0), qml.PauliY(1)),
             qml.s_prod(2.0, qml.PauliX(0)),
             qml.Hamiltonian([1.0, 2.0], [qml.PauliX(0), qml.PauliY(0)]),
+            qml.QubitUnitary(qml.PauliX(0).compute_matrix(), wires=0),
             qml.Hermitian(np.eye(2), wires=0),
             qml.SparseHamiltonian(qml.PauliX.compute_sparse_matrix(), wires=0),
+            qml.QubitUnitary(qml.PauliX.compute_sparse_matrix(), wires=0),
```

Comment:
We cannot use QubitUnitary as observables yet! You need to update yaml files as well as a few other places if it's intended.